upgrade.go

```diff
@@ -36,7 +49,12 @@ func upgrade(cmd *flag.Command) (err error) {
 		return
 	}
 
-	lib := gomu.LibraryFromPath(path.Join(usr.HomeDir, "go", "src", "github.com", "vroomy", "vpm"))
+	// libraries should be in GOPATH, implementation found in cmd/go/internal/cfg/cfg.go, `
+	goPath, err := getGoPath()
```

Comment:
Can you declare goPath in the `var` block above? Then you can remove duck typing.

```go
if goPath, err = getGoPath(); err != nil {
    return
}
```

Reply:
Done!